content/docs/next-release/operator.md

```diff
@@ -391,6 +391,22 @@ storage:
     image: jaegertracing/jaeger-es-index-cleaner  // image of the job
 ```
 
+## Spark dependencies
+
+Spark dependencies collects spans from storage, analyzes links between services and stores them for later presentation in the UI.
```

Comment:
"Spark dependencies collects" -> "The processing to derive dependencies will collect ..."
"and stores" -> "and store"